BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/physicsOfResultsPUEO.py

```diff
@@ -0,0 +1,246 @@
+
+                    TotalEvents[energy].append(allEvents)
+                    passTree = IceFinalFile.passTree
+                    passEvents = passTree.GetEntries()
+                except:
+                    print("Error opening file")
+                    continue
+                success_runs.append(fileName)
+                
+                for i in range(0,passEvents):
+                    j = j + 1
+                    
+                    passTree.GetEvent(i)
+                    allTree.GetEvent(i)
+                    
+                    # Part from rootAnalysis.py
+                    nuPasses.append(1)
+                    nuWeights.append(passTree.event.neutrino.path.weight
+                                     /(passTree.event.loop.positionWeight
+                                       *passTree.event.loop.directionWeight))
+                    RawWeights[energy].append(nuWeights[-1])
+
+                    # collecting varaibles in the dictionary
+
+                    trigg = j
+                    var_dict['trigg'].append(j)
+                    passed = 1
+                    passWeight = nuWeights[-1]
+                    rawWeights = nuWeights[-1]
+                    position = passTree.event.interaction.position
+                    #pathEntry = passTree.event.neutrino.path.entry doesn't seem to be included
+                    #pathExit = passTree.event.neutrino.path.exit
+                    neutrinoFlavor = passTree.event.neutrino.flavor
+                    interactionLength = passTree.event.interaction.length
+                    interactionCrossSection = passTree.event.interaction.crossSection
+                    interactionStrength = passTree.event.interaction.strength
+                    showerPnuEv = passTree.event.shower.pnu.eV
+                    maxEField = passTree.event.signalSummaryAtDetector.maxEField
+                    maxEFieldFreq = passTree.event.signalSummaryAtDetector.maxEFieldFreq
+                    if(passTree.event.neutrino.flavor==1):
+                        nu_e = 1
+                        nu_m = 0
+                        nu_t = 0
+                    elif(passTree.event.neutrino.flavor==2):
+                        nu_e = 0
+                        nu_m = 1
+                        nu_t = 0
+                    else:
+                        nu_e = 0
+                        nu_m = 0
+                        nu_t = 1
+                    
+                    all_var = [trigg, passed, passWeight, rawWeights,
+                               position, neutrinoFlavor, 
+                               interactionLength, interactionCrossSection, 
+                               interactionStrength, showerPnuEv, maxEField,
+                               maxEFieldFreq, nu_e, nu_m, nu_t]
+                    
+                    for k in range(1, len(all_var)):
+                        var_dict['{0}'.format(var[k])].append(all_var[k])
```

Comment:
```suggestion
                    for k, v in enumerate(all_var, start=1):
                        var_dict[var[k]].append(v)
```
The `'{0}'.format(var[k])` is equivalent to `str(var[k])`, but since they are already strings, it's unnecessary.  Note that if you start from `1`, it will skipp the `trigg`.